SpanContext.IsRemote is false on remote children (#538)
open-telemetry/opentelemetry-specification: specification/api-tracing.md

TraceID and a non-zero SpanID.

`IsRemote` is a boolean flag which returns true if the SpanContext was propagated

> from a remote parent. A child of a remote span should have IsRemote set to false.

Reviewer: ```suggestion
from a remote parent.
When creating children from remote spans, their IsRemote flag MUST be set to false.
```

Otherwise one could interpret this as "The child span's IsRemote flag must always be false if the parent used to be remote, even after the child was propagated.".

Author: good point, thank you!

Reviewer 2: Why only "from remote spans"? I think this part is redundant. Of course when creating children from local spans, their IsRemote flag MUST also be false.

Reviewer 2: Actually, what about spans that are no children at all, but root spans?  Of course their IsRemote flag MUST also be false. I think we should try to find a better way to phrase this.
